media/js/app/taxonomy/taxonomy.js

```diff
@@ -442,31 +448,45 @@
 
             // split the url.
             var urls = value.split(',');
+            var flagArray = [];
```

Comment:
I will have to add the (urls[i].indexOf("json") > -1) check when calling getTheOnomy at line 477 because the test url will not match the regex.